docs/admin/index.rst

```diff
@@ -0,0 +1,154 @@
+   - an `NGINX Ingress Controller  <prerequisites/nginx.html>`_ configured
+   - a `DNS <prerequisites/dns.html>`_ domain for Renku
+
+Optionally, you can choose to manage your own instances of the following:
+
+   - a `Gitlab stand alone instance <https://about.gitlab.com/install/>`_
+   - a Keycloak stand alone instance
+
+B. Pre-deployment steps
+-----------------------
+
+1. (optional) Stand alone configurations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   - `Stand alone Gitlab <configurations/standalone-gitlab.html>`_
+   - `Stand alone Keycloak <configurations/standalone-keycloak.html>`_
+
+2. Create and configure Renku PVs and PVCs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All of Renku information is stored in three volumes: Jena, Postgresql and Gitlab (if not stand alone).
+To this end, persistent volumes and persistent volume claims need to be created. You can use the following yaml files as a base to execute the `kubectl` commands below.
+
+   - `Renku PV yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pv.yaml>`_
+   - `Renku PVC yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pvc.yaml>`_
+
+   .. code-block:: console
+
+     $ kubectl create -f renku-pv.yaml
+     $ kubectl create ns renku
+     $ kubectl -n renku create -f renku-pvc.yaml
+     $ kubectl -n renku edit pvc renku-postgresql
+     $ kubectl get pv
+     $ kubectl get pvc -n renku
+     $ kubectl describe persistentvolumeclaim -n renku  ## this should not show any error, just PVs ready to be used
+
+Ensure that PVs have a `default storageclass` and the following value is set:
+
+   .. code-block:: console
+
+     metadata:
+       name: default
+       annotations:
+         storageclass.beta.kubernetes.io/is-default-class: "true"
+
+3. Create a renku-values.yaml file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This step is very important as it will define the values that helm will use to install your Renku instance.
+
+You can find a basic file in `Renku values file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-values.yaml>`_
+
+Ensure the following has been taken care of before deploying Renku:
+
+  - `variables_switchboard` section has URLs and DNS names that correspond to your configuration
+  - `credentials` section has correctly populated values. Please see the comments in the renku-values file for more information.
+  - check the `resources.requests.memory` value makes sense
+  - `lfsObjects` & `registry` require to setup your S3 back-ends, so you need to have that configuration handy and in place
+
+C. Deploying Renku
+------------------
+
+1. (Optional step for standalone Gitlab)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To be able to support notebooks in private projects we need to insert the gitlab sudo token as a secret so that docker can download the corresponding image from the registry.
```

Comment:
```suggestion
To be able to support notebooks in private projects we need to insert the gitlab admin token as a secret so that docker can download the corresponding private images from the registry.
```

it is _not_ a sudo token - it only needs to have admin registry privileges - a sudo token implies that it can impersonate any user for the API